src/server/src/main/java/io/cassandrareaper/core/RepairRun.java

```diff
@@ -169,14 +181,15 @@ public String toString() {
   public enum RunState {
     NOT_STARTED,
     RUNNING,
+    RUNNING_TF_PAUSED,
```

Comment:
I have mixed feelings about introducing a new run state. Since it's a state that has to be computed from the current time and the pause interval, I think we should refrain from persisting it in the backend, and re-compute it each time it's necessary.
wdyt?